README.md

```diff
@@ -86,46 +87,87 @@ Ensure you have the following installed:
 
 Follow these steps to set up and run the project using Docker.
 
-1. **Build and Start the Container**
+1. **Build and Start the Containers**
 
     ```bash
-    # build the container
-    make docker-build
-    # start the container
-    make docker-start
+    # pull the supporting containers
+    make compose-pull
+
+    # build the main application image
+    make compose-build
+
+    # start the containers
+    make compose-up
     ```
 
 2. **Migrations and Superuser Creation**
 
     If this is your first time setting up the project, you’ll need to run migrations to set up the database schema and create an administrative user.
-    Note: `dj` is an alias for `django-admin`
 
     ```bash
     # ssh into the web container
     make docker-shell
 
     # Run database migrations
-    dj migrate
+    make migrate
 
     # Create a superuser for accessing the admin interface
-    dj createsuperuser
+    make createsuperuser
     ```
 
 3. **Start Django Inside the Container**
 
     Once the containers are running, you need to manually start Django from within the web container. This allows for running both the Django server and any additional background services (e.g., schedulers).
 
+    ⚠️ WARNING  
+    The `honcho` command will pick up your local mounted `.env` file when running via `docker-compose`. Ensure that you comment out any variables in the `.env` file which might cause clashes in the container context as they will take precedence when running `honcho start`.
+
     ```bash
     # Start both Django and the scheduler using Honcho
     honcho start
 
-    # To run just the web server. alias: djrun
     # This is not needed if you used `honcho start`
-    dj runserver 0.0.0.0:8000
+    make runserver
     ```
 
 You can then access the admin at `http://0.0.0.0:8000/admin/` or `http://localhost:8000/admin/`.
 
+#### Running locally with supporting services in Docker
+
+You can also run the main application locally with the supporting backend services such as the Postgres and Redis running in Docker. This can be useful when you want to make changes that require the app to be frequently restarted to be picked up.
+
+In order to run it:
+
+1. Pull the images of the supporting services.
+
+```
+make compose-dev-pull
+```
+
+2. Start the supporting services in Docker.
+
+```
+make compose-dev-up
+```
+
+3. Run the below command to run the necessary pre-run steps:
+
+```
+make dev-init
+```
+
+4. Run the Django server locally via your IDE or with the following command:
+
+```
+make runserver
```

Comment:
The current README has `dj runserver 0.0.0.0:8000` so should it not be `make runserver 0.0.0.0:8000` ? I don't suppose this will have a large effect but thought it be better to point out.

Reply:
The default is 127.0.0.1:8000 [ref](https://docs.djangoproject.com/en/5.1/ref/django-admin/#runserver), so this is fine